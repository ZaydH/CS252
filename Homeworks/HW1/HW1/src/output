[455,1]
[455,3]
[455,235,681]
[455,235,681]
[455,1,681]
[999]
[962,634,9]
[1]
[0]
[12]
[0]
[392,296,4,12]
[256]
[1]
7
63
256
800,000,000,000,000,000,002
1
1,000
53,811,077,339,110
8,471,568,252
256
1,000,000,000,000,000,000,000,000,000,000
884,421,465,464
74,313,467
0
0
5,465,499
1,000,000
1
875,000
10,715,086,071,862,673,209,484,250,490,600,018,105,614,048,117,055,336,074,437,503,883,703,510,511,249,361,224,931,983,788,156,958,581,275,946,729,175,531,468,251,871,452,856,923,140,435,984,577,574,698,574,803,934,567,774,824,230,985,421,074,605,062,371,141,877,954,182,153,046,474,983,581,941,267,398,767,559,165,543,946,077,062,914,571,196,477,686,542,167,660,429,831,652,624,386,837,205,668,069,376
140,725,129,634
874,152,625,849
0
