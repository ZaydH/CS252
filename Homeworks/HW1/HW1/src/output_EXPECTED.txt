[455,1]
[455,3]
[455,235,681]
[455,235,681]
[455,1,681]
[999]
[962,634,9]
[1]
[0]
[12]
[0]
[392,296,4,12]
[256]
[1]
7
63
256
800,000,000,000,000,000,002
1
